source/cppfront.cpp

```diff
@@ -2709,4 +2711,5 @@ auto main(int argc, char* argv[]) -> int
             c.debug_print();
         }
     }
+    return ret_status;
```

Comment:
I understand the C++ standards only allow for 0 (EXIT_SUCCESS) and 1 (EXIT_FAILURE).
In POSIX, only the least significant byte of ret_status would be used.
So, -1 and -2 are potentially problematic for those reasons.

https://en.cppreference.com/w/cpp/utility/program/exit
https://pubs.opengroup.org/onlinepubs/007904875/functions/exit.html

Reply:
thanks for notice, updated to EXIT_SUCCESS and EXIT_FAILURE